tools/llm_bench/llm_bench_utils/ov_utils.py

```diff
@@ -249,6 +252,12 @@ def convert_ov_tokenizer(tokenizer_path):
 
 def create_image_gen_model(model_path, device, **kwargs):
     model_class = IMAGE_GEN_CLS
```

Comment:
there are also models that supported only by inpainting class, can we handle this case and avoid selecting for them text2image pipeline. You can understand that based on model_index.json, they will have Inpaint suffix in _class_name parameter, e.g. https://huggingface.co/stabilityai/stable-diffusion-2-inpainting/blob/main/model_index.json#L2

Reply:
added condition for suffix